Tests/web3swiftTests/localTests/ArrayExtensionTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ArrayExtensionTests.swift
+//  Created by albertopeam on 25/11/22.
+//
+
+@testable import Web3Core
+import XCTest
+import BigInt
+
+final class ArrayExtensionTests: XCTestCase {
+    func testToAnyObjectEmpty() {
+        let result = [].toAnyObject()
+        XCTAssertEqual(result.count, 0)
+    }
+
+    func testToAnyObjectNils() throws {
+        let result = [nil, nil].toAnyObject()
+        XCTAssertEqual(result.count, 2)
+        XCTAssertTrue(result.first is NSNull)
+        XCTAssertTrue(result.dropFirst().first is NSNull)
+    }
+
+    func testToAnyObjectNilAndNonNils() throws {
```

Comment:
In addition I would add a test case where this method fails to accept value an object casted to any.

Reply:
Not sure exactly what you mean. AFAIK, anything can be cast to `AnyObject`.

Reply:
Yep, as you've provided details above that case is unreachable.